caput/config.py

```diff
@@ -510,7 +510,7 @@ class Project:
 
             mode = file_format(default='zarr')
     """
-    options = ("hdf5", "zarr", None)
```

Comment:
without a `None` default option, the `caput.fileformats.check_file_format` guessing functionality is broken.

Reply:
@tristpinsm is looking into this.